src/main/java/Duke.java

```diff
@@ -1,10 +1,234 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_LIST_SIZE = 100;
+    public static final String CHAT_BOT_NAME = "Arthur";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greet();
+        processCommands();
     }
+
+    private static void processCommands() {
+        String line;
+        List dukeList = new List(CHAT_BOT_NAME);
+        Todo todoList = new Todo(MAX_LIST_SIZE);
+        Deadline deadLineList = new Deadline(MAX_LIST_SIZE);
+        Event eventLists = new Event(MAX_LIST_SIZE);
+
+        Scanner in = new Scanner(System.in);
+
+        boolean hasToContinue = true;
+        dukeList.printHelp();
+        while(hasToContinue){
+            line = in.nextLine();
+            hasToContinue = selectCommand(dukeList,line, hasToContinue,todoList,deadLineList,eventLists);
```

Comment:
Inconsistent spacing before and after commas.

Reply:
Noted!!!